client/src/assets/js/app/typing-lesson.js

```diff
@@ -113,6 +115,22 @@
       isRunning: function() {
         return (this.state === LevelStates.Running);
       },
+      incorrectCharsSorted: function() {
+        let charsCount = this.result.missed_chars_count;
+
+        for(let key in charsCount){
```

Comment:
We should assign `charsCount[key]` to a variable since it's used multiple times.